rllib/evaluation/rollout_worker.py

```diff
@@ -48,7 +48,7 @@
 
 if TYPE_CHECKING:
     from ray.rllib.evaluation.observation_function import ObservationFunction
-    from ray.rllib.agents.callbacks import DefaultCallbacks
+    from ray.rllib.agents.callbacks import DefaultCallbacks  # noqa
```

Comment:
What is #noqa?

Reply:
:) makes the LINTer not complain about it (e.g. DefaultCallbacks) being imported, but then "only" used in quotes in the code below. We have to use quotes sometimes to avoid circular import dependencies.